aspnet/migration/rc1-to-rc2.rst

```diff
@@ -28,64 +28,213 @@ You must update the SDK version in ``global.json``, as this file is used to conf
+        "version": "1.0.0-preview2-final",
+        "imports": [ "portable-net45+win8+dnxcore50" ]
+      }
+    }
+  }
+
+See `this document <https://docs.efproject.net/en/latest/cli/dotnet.html>`_ for more information on the ``ef`` .NET Core CLI tool.
+
+Razor tools
+^^^^^^^^^^^
+
+Likewise, Razor tooling is now provided in the ``Microsoft.AspNetCore.Razor.Tools`` package:
+
+.. code-block:: json
+
+  {
+    "tools": {
+      "Microsoft.AspNetCore.Razor.Tools": {
+        "version": "1.0.0-preview2-final",
+        "imports": [ "portable-net45+win8+dnxcore50" ]
+      }
+    }
+  }
+
+SQL cache tool
+^^^^^^^^^^^^^^
+
+The ``sqlservercache`` command, formerly provided by the ``Microsoft.Extensions.Caching.SqlConfig`` package, has been replaced by the ``sql-cache`` .NET Core CLI tool, available through the ``Microsoft.Extensions.Caching.SqlConfig.Tools`` package:
+
+.. code-block:: json
+
+  {
+    "tools": {
+      "Microsoft.Extensions.Caching.SqlConfig.Tools": {
+        "version": "1.0.0-preview2-final",
+        "imports": [ "portable-net45+win8+dnxcore50" ]
       }
     }
   }
 
-The default server URL and port are ``localhost:5000``. You can find more information about Garbage Collection configuration at: https://github.com/aspnet/Announcements/issues/175
+User secrets manager
+^^^^^^^^^^^^^^^^^^^^
+
+The ``user-secret`` command, formerly provided by the ``Microsoft.Extensions.SecretManager`` package, has been replaced by the ``user-secrets`` .NET Core CLI tool, available through the ``Microsoft.Extensions.SecretManager.Tools`` package:
+
+.. code-block:: json
+
+  {
+    "tools": {
+      "Microsoft.Extensions.SecretManager.Tools": {
+        "version": "1.0.0-preview2-final",
+        "imports": [ "portable-net45+win8+dnxcore50" ]
+      }
+    }
+  }
+
+File watcher
+^^^^^^^^^^^^
+
+The ``watch`` command, formerly provided by the ``Microsoft.Dnx.Watcher`` package, has been replaced by the ``watch`` .NET Core CLI tool, available through the ``Microsoft.DotNet.Watcher.Tools`` package:
+
+.. code-block:: json
+
+  {
+    "tools": {
+      "Microsoft.DotNet.Watcher.Tools": {
+        "version": "1.0.0-preview2-final",
+        "imports": [ "portable-net45+win8+dnxcore50" ]
+      }
+    }
+  }
+
+Runtime environment information
+-------------------------------
+
+The ``RuntimeEnvironment`` class is no longer available in the ``Microsoft.Extensions.PlatformAbstractions`` package. You should read runtime environment information from ``System.Runtime.InteropServices.RuntimeInformation``.
+
+Hosting
+-------
+
+Creating the web application host
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-All classes prefixed with WebApplication have been renamed to WebHost. This includes:
+Since ASP.NET Core 1.0 apps are just console apps, you must define an entry point for your application that sets up a web host and runs it. Below is an example from the startup code for the `Web Application` template in Visual Studio:
```

Comment:
You create a AspNetCore Web Application project and then choose between EmptyWeb, WebAPI and WebApplication right?